### ISSUE TITLE
main():225 SRR unexpected behavior: _Map_base::at

### ISSUE THREAD

**Issue by symeonp** [tetrane/reven-community#12]
**Describe the bug**
Since the replay is partially saved, starting Axion (or the server clearly fails)...
but my question would be, why the replay is not succeeding?

Trouble is, I've tried on both Ubuntu 20 as well as Debian (following the tutorial on how to create a new scenario)
on both Mac OS and Windows host and no matter what it kept stuck with exactly the above partial replay. Not sure what am I missing?

**To Reproduce**
1. Create the Win7 VM snapshot as per tutorial and follow the guide on ['Record a first scenario](https://doc.tetrane.com/free/latest/Quick-Start/Record.html)'
Again the issue lies within the fact that the step 2 is NOT complete:

"Wait for all resources to be done and at 100%. This will take a few minutes."

**Expected behavior**
Complete the Replay 

**Screenshots**
https://i.snipboard.io/YIWyL7.jpg

**Host Environment**
- REVEN Version 2.10.1
- REVEN Edition Free Edition
- REVEN Package: Docker
- VMWare ESXi  is the hypervisor running on an HP MicroServer 

**Conditions of occurrence**
- Record is performed as per documentation 
- Target environment Windows 7 x86

Please note that for the target VM I *have not* enabled UEFI as it wouldn't boot.

**Logs**
```
Start date: 2021-12-14 20:35:34.371597+00:00
Command: reven_backend --port 0 --trace /Reven2/2.10.1/Replays/show-hostname_802122b0-b224-4b3a-8358-b407e03278c3 --user-data /Reven2/2.10.1/Scenarios/show-hostname_802122b0-b224-4b3a-8358-b407e03278c3/UserData --name 'Show hostname' --filesystem /Reven2/2.10.1/Mountpoints/sessions/348a3427-847d-4791-850e-5f06ac51696a --os Windows-x86 --symstore /home/reven/.local/share/reven/symbols
[0;34m PdbManager():65 [0;37m[0;44;37mTRC[0m [0mOSSI: SYMSTORE provided "/home/reven/.local/share/reven/symbols".[0m
[0;34m load_windows_x86_kernel_description():448 [0;30m[0;43;30mWAR[0m [0;33mOssiException: Error while loading kernel description file: No such node (structures._KPCR.members.PrcbData)[0m
[0;34m load_windows_x86_kernel_description():449 [0;30m[0;43;30mWAR[0m [0;33mFallback on filesystem and symbols to load kernel description.[0m
[0;34m main():225 [0;30m[0;41;30mSRR[0m [0;1;31munexpected behavior: _Map_base::at[0m
```

**Additional context**
Obviously since the OSSI Ranges have not been fully saved this yields another error
```
[2021-12-14 20:18:35,955][INFO] Log started at 2021-12-14 20:18:35.955669+00:00
[2021-12-14 20:18:35,955][INFO] Log into /Reven2/2.10.1/Logs/Tasks/f0c3ca0b-8b35-49d3-8d04-bb5faaa467e1.log
[2021-12-14 20:18:35,955][ERROR] Failed because the generation of 'OSSI Ranges' failed
```

**Comment by johdcmlaker2000** [comment]
I had the same error when I tried to run the x64 OS image and specifyed image as x86

**Comment by johdcmlaker2000** [comment]
try to 

1. From any page of the Project Manager, click "VM Manager"
2. Click on the name of your VM in the VM list.
3. Click "Edit VM preferences"
4. Change the "Archi" field to "x64"
5. Go in Scenario Manager, click the "Replay" button next to the name of your scenario
6. Delete the resources associated with the OSSI feature, then replay them again
7. Try starting again the REVEN server on your scenario

**Comment by johdcmlaker2000** [comment]
the problem is that the links to download images from the microsoft offsite, we download x86 image of windows7, but in fact it loads windows 7 x64 image (the problem is in the links on the microsoft offsite)

**Comment by symeonp** [comment]
Hey, thank you **so** much!! Unbelievable, this did indeed work!! I can **finally** play with Axion!
I would have never thought of changing the architecture as this would probably mess up my VM, apparently that's not the case, did you experiment with that?

Thanks again, much much appreciate it :)

**Comment by Quentin01** [comment]
Hi,

It's great to see REVEN's users helping each others with their issues.

This issue have a fix in the 2.10.2 that will soon be available for everyone (including the free edition).
